simd/unit_tests/include/TestSIMD_MathOps.hpp

```diff
@@ -20,6 +20,7 @@
 #include <Kokkos_Macros.hpp>
 #ifdef KOKKOS_ENABLE_EXPERIMENTAL_CXX20_MODULES
 import kokkos.simd;
+import kokkos.simd_impl;
```

Comment:
This import might be needed in other simd unit test files as well, since they all uniformly use definitions from the `Impl` namespace.